### DISCUSSION TITLE
Why swastika on oh my zsh theme ????

### DISCUSSION THREAD

**Discussion by volga629-1** [ohmyzsh/ohmyzsh#12263] [Help] [answered]
On main web site posted theme with swastika ???

<img width="2067" alt="Screenshot 2024-03-06 at 9 21 46 AM" src="https://github.com/ohmyzsh/ohmyzsh/assets/59034879/c6c30780-0cbd-419b-b382-7a5353676dca">

**Comment by mcornella** [comment, marked as answer]
Hello @volga629-1, thanks for the report.

We take extremist views on our project very seriously and we use moderation to ensure that we don't give a platform to them.

Fortunately in this case, the symbols in the external theme documented in our wiki [1] belong to a religion.

Swastikas have long been a symbol of good fortune in several Asian religions [2], many hundreds (and possibly thousands) of years before nazism co-opted them. Unfortunately the symbol is now most known for nazism, but it is still very widely used among countries professing such religions [2].

I have added a disclaimer in [1] for future reference. 

[1] https://github.com/ohmyzsh/ohmyzsh/wiki/External-themes#amresh-terminal-theme

[2] https://en.wikipedia.org/wiki/Swastika

**Reply by volga629-1** [reply]
Thank you for explanation.
